code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -318,12 +318,12 @@
 			module_sprites["Basic"] = "secborg"
 			module_sprites["Blue Knight 2.0"] = "sleeksecurity"
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
 			module_sprites["Securitron"] = "securitron"
-			module_sprites["Marina-SC"] = "marinaSC"
+			module_sprites["Marina-PK"] = "marinaSC"
 			module_sprites["#9"] = "servbot-sec"
 			module_sprites["Kodiak"] = "kodiak-sec"
-			to_chat(src, "<span class='warning'><big><b>Just a reminder, by default you do not follow space law, you follow your lawset</b></big></span>")
+			to_chat(src, "<span class='warning'><big><b>You are expected to follow orders and prevent harm above all else. Space law means nothing to you- you are not a Security Officer.</b></big></span>")
```

Comment:
>You are expected to follow orders and prevent harm above all else.

Really only applies to asimov. Replace it with something like "You are expected to follow orders and keep the peace" or add a per-law clause, this is dumb.